cybergis_compute_client/UI.py

```diff
@@ -507,6 +545,31 @@ def on_change(change):
                 self.rerender(['description', 'slurm', 'param', 'uploadData'])
         return on_change
 
+    def onLoadMoreClick(self):
+        def on_click(change):
+            self.recently_submitted['job_list_size'] += 5
+            getattr(self, 'recently_submitted')['output'].clear_output()
+            getattr(self, 'load_more')['output'].clear_output()
+            self.renderRecentlySubmittedJobs()
+            self.renderLoadMore()
+        return on_click
+
+    def onJobEntryButtonClick(self, job_id):
+        def on_click(change):
+            job = self.compute.get_job_by_id(job_id, printJob=False)
+            self.compute.job = job
+            self.jupyter_globus = self.compute.get_user_jupyter_globus()
+            self.globus_filename = 'globus_download_' + self.compute.job.id
+            self.tab.selected_index = 2
+            self.submitted = True
+            self.tab.set_title(2, '⏳ Your Job Status')
+            self.rerender(['resultStatus', 'resultEvents', 'resultLogs', 'submit'])
+            getattr(self, 'recently_submitted')['output'].clear_output()
```

Comment:
Any particular reason you used these 

>getattr(self, <constant string>)

instead of just

>self.recently_submitted['output']

I don't think I've ever used getattr() before unless the attribute was a variable like in this line: https://github.com/cybergis/cybergis-compute-python-sdk/blob/v2/cybergis_compute_client/UI.py#L552